_sass/_base.scss

```diff
@@ -282,6 +282,22 @@ footer.sticky-bottom {
   font-size: 0.9rem;
 }
 
+// CV
+
+.cv {
+  .card {
+    background-color: var(--global-code-bg-color);
```

Comment:
This was intentional, since it looks better with the `code-bg-color`.

And also, the cards are indistinguishable with the background if both have the same color.

Reply:
The cards still have a shaded frame, which sets them apart.

At least in al-folio's default color scheme,I don't think it works (it has black text on a dark blue background). In dark mode, it kinda pops. Maybe it comes down to taste in the end and how you set up your color scheme, but to me, it looked faulty.

Reply:
I found that there was another variable named `--global-card-bg-color`, I felt it was more suited, so I changed the card colors to that.